### DISCUSSION TITLE
Can I use 2 hasura  on the same database?

### DISCUSSION THREAD

**Discussion by chanthavong** [hasura/graphql-engine#7705] [Question]
currently I use v1.3.0 function aggregate sometime the summary is wrong, so can I use the current version  to use function aggregate in  the same database . the reason why I want to use 2 hasura in the same db because my current project does not support the current version

**Comment by dmi3y** [comment]
I'm trying to archive the similar goal and I believe this is not possible, at the moment,  due to `hbd_catalog` schema being reused by v2 (I'm trying to connect v2 with separate metadata DB to the DB which has connected v1)... though looking at "purely" `hbd_catalog` from v2 on data DB (not metadata DB) it looks pretty minimal comparing to v1.

That would be handy to be able connect different versions without overlapping.
